landing/src/pages/Home/LiqiuditySection/PoolCards.tsx

```diff
@@ -22,7 +22,7 @@ export function PoolCards() {
   const onClickGm = useGoToPools("GM");
   const { poolsData } = useHomePageContext();
   const processedData = useProcessedData(ARBITRUM);
-  const gmxAprForGmxPercentage = Number(processedData?.gmxAprForGmx) / 10 ** DECIMALS;
+  const gmxAprForGmxPercentage = Number(processedData?.data?.gmxAprForGmx) / 10 ** DECIMALS;
```

Comment:
not related to this PR but probably let's make it more readable without 10 ** DECIMALS?